format_strawberryfield.routing.yml

```diff
@@ -65,6 +65,23 @@ format_strawberryfield.iiifbinary:
     format: .+
     _entity_access: 'node.view'
 
+# Direct File access for SBF managed files via UUID.
+format_strawberryfield.iiifbinary_uuid:
```

Comment:
Hi, even if for legacy reasons the controller class is named IiifBinaryController, it would be better to name the route simply format_strawberryfield.binary maybe?